pxtblocks/fields/field_ledmatrix.ts

```diff
@@ -435,6 +435,10 @@ export class FieldMatrix extends Blockly.Field implements FieldCustom {
         return `\`\n${FieldMatrix.TAB}${text}\n${FieldMatrix.TAB}\``;
     }
 
+    getFieldDescription(): string {
+        return lf("LED Grid");
```

Comment:
nitpick: maybe include the dimensions? e.g. "5x5 LED Grid". the microbit has a couple blocks in the images category with different grid sizes